code/obj/item/clothing/masks.dm

```diff
@@ -380,6 +380,12 @@ TYPEINFO(/obj/item/clothing/mask/monkey_translator)
 			src.item_state = "clown_hat"
 			user.show_text("You untuck the wig from the [src].")
 
+	autumn
+		name = "autumn clown wig and mask"
+		desc = "A special clown mask made to celebrate Autumn. Orange you glad you have it!!"
+		icon_state = "clown_autumn"
+		item_state = "clown_autumn"
```

Comment:
Inconsistency bad. I do prefer `clown_autumn` over `autumn_clown` by a lot tho (kinda like color is always after).

Reply:
did this (changed every other autumn clown item to use clown_autumn), though note that the way I had it set up was consistent *in the files they were in* so they're inconsistent with other clown items in that way. but then the clown items are inconsistent with themselves so :shrug: